### PR TITLE
change directory hierarchy

### PR DESCRIPTION
**Pull request by kdada**
- fix README.md typo
- decrease directory hierarchy
- add packages
- change image path to cargo.caicloudprivatetest.com/library/*

### CODE REVIEW

**Comment by ddysher** [comment]
LGTM

@ijumps

File: stable/mongodb/0.4.7/values.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ _config:
   mongodb:
     containers:
       mongodb:
-        image: "bitnami/mongodb:3.4.2-r0"
+        image: "cargo.caicloudprivatetest.com/library/mongodb:3.4.4-r5"
```
Comment (ddysher) [inline stable/mongodb/0.4.7/values.yaml]: We should use public repo by default

Comment (ddysher) [inline stable/mongodb/0.4.7/values.yaml]: not a priority right now

**Comment by ijumps** [comment]
LGTM

**Pull request merged**